### PR TITLE
Review

### PR DESCRIPTION
**Pull request by FeasibleSnail85**

### CODE REVIEW

**Review by Mr-X1214 — COMMENTED** [review]
In the die class the names stick to the naming conventions in the style guide and the comments are useful but could explain things a bit more. You should probably create the Random variable in the method Roll() also it doesn't need to be static.

In the game class you should put more comments in to fully explain things in the code for better understanding for people that read the code. The names fit in with the naming conventions in the style guide. The creation of the dice objects could be put into the method.

In the program class the names fit in with the naming conventions. The comments could be a bit more descriptive. 

In the testing class you probably should add more comments to explain things in the code that aren't that obvious. For the Debug.Assert() where you test the sum of the dice, I believe it's supposed to be testing that it's within a certain range, so basically what has been done for the dice roll test. Naming conventions have been stuck to.

In summary, the code is easy to read but not necessarily easy to understand why you've done specific things, that could be due to the lack of comments though. The requirements of the brief have been met.
